File: 944 - Delete Columns to Make Sorted.java
class Solution {
    public int minDeletionSize(String[] strs) {
        String[] res = new String[strs[0].length()];
        int count = 0;
        for (int i = 0; i < res.length; i++) {
            res[i] = "";
            for (int j = 0; j < strs.length - 1; j++) {
                if (strs[j].charAt(i) > strs[j + 1].charAt(i)) {
                    count++;
                    break;
                }
            }
        }
        return count;
    }
}

File: 944 - Delete Columns to Make Sorted.py
class Solution:
    def minDeletionSize(self, strs: list[str]) -> int:
        res = []
        count = 0
        for i in range(len(strs[0])):
            temp = ''
            for s in strs:
                temp += s[i]
            res.append(temp)
        for s in res:
            if s != ''.join(sorted(s)):
                count += 1
        return count